src/pfe/portal/routes/projects/index.js

```diff
@@ -28,7 +28,8 @@ const router = express.Router();
   require('./fileChanges.route'),
   require('./remoteBind.route'),
   require('./links.route'),
+  require('./missingLocalDir.route')
 ]
   .forEach((subRouter) => router.use(subRouter));
 
-module.exports = router;
+module.exports = router;
```

Comment:
This and missingLocalDir.route.js are both missing newlines at the end of the files. Might be worth fixing.